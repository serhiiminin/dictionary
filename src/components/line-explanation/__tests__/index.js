import React from 'react';
import renderer from 'react-test-renderer';
import LineExplanation from '..';


describe('Line explanation', () => {
  const component = renderer.create(
    <LineExplanation
      label='label'
    >text</LineExplanation>
  );
  const tree = component.toJSON();

  test('render', () => {
    expect(tree).toMatchSnapshot();
  });
});
