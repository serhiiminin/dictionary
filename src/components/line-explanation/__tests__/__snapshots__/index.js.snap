// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line explanation render 1`] = `
<div
  className="LineExplanation-lineExplanation-0-1-1"
>
  <div
    className={undefined}
  >
    label:
  </div>
  <div
    className={undefined}
  >
    text
  </div>
</div>
`;
